eng/testing/tests.mobile.targets

```diff
@@ -101,14 +101,14 @@
     </RuntimeConfigParserTask>
 
     <MonoAOTCompiler Condition="'$(RunAOTCompilation)' == 'true'"
-        CompilerBinaryPath="$(MicrosoftNetCoreAppRuntimePackNativeDir)cross\$(PackageRID)\mono-aot-cross"
+        CompilerBinaryPath="$(MonoArtifactsPath)\cross\$(PackageRID)\mono-aot-cross"
```

Comment:
Would it be possible for the AOT compiler to ship with an MSBuild file that sets a variable with the path to the AOT compiler? That way it wouldn't have to be hard-coded everywhere.

Reply:
I found that `$(MonoAotCrossCompilerPath)` was set when I tried the packages here:

https://github.com/xamarin/xamarin-android/pull/5539/files#diff-eeb0bf74a53216ade5ce332b4a0af964f3eb06e2a83a9e2b59d6b915f308e5e0R86

This PR isn't done -- it's not working in its current state.

Reply:
@rolfbjarne `$(MonoAotCrossCompilerPath)` should be set for you when you reference the aot package.